parfive/downloader.py

```diff
@@ -85,33 +85,22 @@ def __init__(self, max_conn=5, progress=True, file_progress=True,
             self.headers = {'User-Agent': f"parfive/{parfive.__version__} aiohttp/{aiohttp.__version__} python/{sys.version[:5]}"}
 
     def _start_loop(self, loop):
-        # Setup asyncio loops
-        if not loop:
-            aio_pool = ThreadPoolExecutor(1)
```

Comment:
The reason the thread pool executor existed was to support running parfive from within a jupyter notebook:
![image](https://user-images.githubusercontent.com/1391051/92139554-d0979c00-ee07-11ea-94ec-5f32403adfae.png)

I am not sure what the best solution is to this, because otherwise this would be much nicer I feel.

Reply:
https://stackoverflow.com/questions/55409641/asyncio-run-cannot-be-called-from-a-running-event-loop might have some answers - I don't have time to look into this at the moment, but might leave the PR open as a reminder for later.

Reply:
I have a working solution, I will push a commit.